tests/fuzzers/ucl_msgpack_fuzzer.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+#include <errno.h>
+#include <unistd.h>
+#include "ucl.h"
+#include "ucl_internal.h"
+#include <ctype.h>
+
+typedef ucl_object_t* (*ucl_msgpack_test)(void);
+
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+	
+	if(size<3){
+		return 0;
+	}
+
+	struct ucl_parser *parser;
+	
+	ucl_object_t *obj = ucl_object_new_full (UCL_OBJECT, 2);
+	obj->type = UCL_OBJECT;
```

Comment:
Here is your leak btw. Why do you create this object that never got used/accessed or freed?